scale/docs/architecture/jobs/resources.rst

```diff
@@ -0,0 +1,54 @@
+
+.. _architecture_jobs_resources:
+
+Resources
+========================================================================================================================
+
+The resources document is a JSON document that defines a set of named resources with floating point values.
+
+**Example job configuration:**
+
+.. code-block:: javascript
+
+  {
+     "version": "1.0",
+     "resources": {
+        "cpus": 1.5,
```

Comment:
Makes sense, just wanted clarification there. I like the general idea of pulling these into a `resource` object in the interest of future extensibility over specific top level keys.